example/my-app/src/Main.purs

```diff
@@ -0,0 +1,12 @@
+module App.Main where
+
+import Prelude
+
+import Data.Either as Either
+import Effect (Effect)
+import Effect.Console (log)
+import Lib as Lib
+import Registry.License as License
+
+main :: Effect Unit
+main = log $ Lib.pizza <> " " <> Either.either identity License.print (License.parse "MIT")
```

Comment:
The flake checks verify that this does print and exit, as can be seen in:
https://github.com/thomashoneyman/purescript-nix/pull/7/checks?check_run_id=13989957830